tests/testthat/test-utils_tfrmt.R

```diff
@@ -217,8 +221,15 @@ test_that("Check apply_tfrmt for mock data",{
   )
   mock_dat <- make_mock_data(plan, .default = 1:2, n_col = 2)
 
-  expect_message(mock_dat %>% apply_tfrmt(plan, mock =TRUE),
-                "Mock data contains more than 1 param per unique label value. Param values will appear in separate rows.")
+  make_mock_dat_message <- mock_dat %>%
+    apply_tfrmt(plan, mock =TRUE) %>%
+    capture_messages()
+
+  ## capturing second message
+  expect_equal(
+    make_mock_dat_message[2],
```

Comment:
I see that we are also keeping the regular frmt_combine msg for mocks. 
If someone creates mocks w/o data, they are unable to say if multiple params live on the same row, in different columns. This would be unrelated to frmt_combine but you'd still receive the frmt_combine suggestion. So, in the case of mocks should we have more general messaging?

Reply:
That is a good point!

Reply:
because we allow users to provide their own data, this is a little difficult to blindly not return the message whether they need to correct the bodyplan for multiple param/cell

Reply:
I just set it to not return those messages when its running a mock, but we may want to think more about it going forward